src/ontology/uberon-edit.obo

```diff
@@ -224906,6 +224906,35 @@ intersection_of: part_of UBERON:0003889 ! fallopian tube
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-21T08:55:11Z xsd:dateTime
 
+[Term]
+id: UBERON:8480038
+name: meningeal myeloid tissue
+def: "Lymphomyeloid tissue that consist mainly in granulopoietic cells but that also contains erythroid and lymphoid cells in different stages of maturation." [PMID:25130144]
+is_a: UBERON:0034769 ! lymphomyeloid tissue
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: in_taxon NCBITaxon:7898 ! Actinopterygii
+property_value: curator_notes "May be present in Amia and Lepisosteus, Polyodon and Acipenser, but not Polypterus." xsd:string {xref="https://doi.org/10.1111/j.1096-3642.1998.tb02524.x"}
+property_value: dcterms-date 2022-10-31T07:50:56Z xsd:dateTime
+
+[Term]
+id: UBERON:8480039
+name: milt
+def: "Seminal fluid of fish, mollusks, and certain other water-dwelling animals which reproduce by spraying this fluid, which contains the sperm, onto the egg cells." [https://en.wikipedia.org/wiki/Milt]
```

Comment:
We like to have a genus-differentia definition, where the genus is an existing term.

Here we might write as

"Seminal fluid which <differentiating feature>"

Where the genus is an is-a parent or ancestor

Here we run into a challenging issue. We have "seminal fluid" in uberon already, you opted not to use it in your is-a, likely wisely, as the existing term may be vertebrate specific (not totally clear). But we should be consistent in terminology and use the same term in your is-a that is in your definition

Reply:
Wikipedia is sometimes a decent source proxy but in this case the wikipedia page is not really helpful and is more about culinary aspects.

To what extend is a new concept really required here vs just having a synonym on seminal fluid?

Having terms that group across phyla is often a sign we want two different concepts, or it's simply different nomenclature on different species for a more general concept.

cc @ddooley

Reply:
@cmungall I let you decide what is more correct:
1) add as synonym of seminal fluid. 
Is possible to indicate that is a synonym/term used in fish?
2) create a new concept. 
Is this def. better?
"Secretion from the glands of the male genital tract containing sperm that is sprayed onto the egg cells in certain water-dwelling animals."
Difficult to find a good source of definition. What about PMID:31287811 ?

Reply:
@cmungall I found that this term is also used in _Petromyzon marinus_ (PMID:31287811), so I think the best is to add "milt" as synonym of seminal fluid